### PR TITLE
Pub 1292 add tand cs to media accounts

### PR DESCRIPTION
**Pull request by furnivall**
### JIRA link ###
https://tools.hmcts.net/jira/browse/PUB-1292

### Change description ###
- Added t&c heading 
- paragraphs 
- new hint consent text 
- error messages if box is unchecked (see image).
image
image
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by joshblackmoor — DISMISSED** [review]

File: src/main/views/create-media-account.njk [hunk 1]
```diff
@@ -122,8 +134,14 @@
         {{ employerInput }}
         <p class="govuk-body" tabindex="0">{{ photoMessage }}</p>
         <p class="govuk-hint" tabindex="0">{{ photoSubMessage }}</p>
-        <p class="govuk-hint" id="photoType" tabindex="0">{{ photoType }}</p>
+        <p class="govuk-hint" tabindex="0">{{ uploadConsent }}</p>
```
Comment (ChrisS1512) [inline src/main/views/create-media-account.njk:137]: Formattings slightly out on this section (just needs making in-line)

File: src/main/views/create-media-account.njk [hunk 2]
```diff
@@ -3,6 +3,7 @@
 {% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
```
Comment (ChrisS1512) [inline src/main/views/create-media-account.njk:3]: E2E tests will need updating to account for the checkbox, as we create a media account as part of them

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/views/create-media-account.njk (hunk 2 above)
Comment (furnivall) [reply src/main/views/create-media-account.njk:3]: This has now been done :)

File: src/main/views/create-media-account.njk (hunk 1 above)
Comment (furnivall) [reply src/main/views/create-media-account.njk]: should be fixed oto

File: package.json [hunk 3]
```diff
@@ -48,6 +48,7 @@
     "axios": "^0.21.4",
     "axios-oauth-client": "^1.4.4",
     "axios-token-interceptor": "^0.2.0",
+    "busboy": "^1.6.0",
```
Comment (ChrisS1512) [inline package.json:51]: This can now be removed :)

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (furnivall) [reply package.json:51]: can do!

**Review by ChrisS1512 — APPROVED** [review]

**Review by joshblackmoor — APPROVED** [review]

**Pull request merged**